{"uuid": "8f69a2d5-7dee-443c-87ce-5c578f083fad", "children": ["caa319d0-5fbd-47d8-aa93-67f75daf2445"], "befores": [{"name": "test_config", "status": "broken", "statusDetails": {"message": "pydantic_core._pydantic_core.ValidationError: 2 validation errors for Config\nSERVICE\n  Field required [type=missing, input_value={}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.11/v/missing\nCARDS\n  Field required [type=missing, input_value={}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.11/v/missing\n", "trace": "  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\tests\\conftest.py\", line 26, in test_config\n    return config.get()\n           ^^^^^^^^^^^^\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\tests\\config.py\", line 26, in get\n    return Config()\n           ^^^^^^^^\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\.venv\\Lib\\site-packages\\pydantic_settings\\main.py\", line 176, in __init__\n    super().__init__(\n  File \"C:\\Users\\FATE\\PycharmProjects\\TETS_FTP_GAMES_LIST\\.venv\\Lib\\site-packages\\pydantic\\main.py\", line 253, in __init__\n    validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1750000890250, "stop": 1750000890250}], "start": 1750000890250, "stop": 1750000890440}